gensim/utils.py

```diff
@@ -266,6 +266,10 @@ def load(cls, fname, mmap=None):
         obj = unpickle(fname)
         obj._load_specials(fname, mmap, compress, subname)
         logger.info("loaded %s", fname)
+        if cls.__name__ is 'Similarity':
```

Comment:
@piskvorky  should issubclass be called from inside a ```classmethod``` function, when we are not calling it from the class we intend to compare it with 

```isinstance(cls, Similarity)``` throws error global name 'Similarity' is not defined